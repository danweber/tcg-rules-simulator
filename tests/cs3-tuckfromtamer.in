VERSION:2

# [Your Turn] When one of your green Monster would be played, by placing 1 card from under one of your Tamers as the bottom digivolution card of that Monster, reduce the play cost by 1.

FIELD1:mon,CS3-Sorcery Tamer,CS3-Birdwatcher,CS3-02,CS3-03 evo,CS3-02

HAND1:CS3-06
GAME:P1 T3 MAIN 6
DUMP

ECHO: ------ play,  should interrupt
CMD:EVO CS3-06 evo
EFFECT-TREE
GET-CHOICES
DUMP

ECHO: -- choose to tuck
STEP:TO:0
EFFECT-TREE
GET-CHOICES
DUMP

ECHO: -- choose chirp, and we evo for 4-1 or 3
STEP:TO:1
EFFECT-TREE
GET-CHOICES
DUMP

EXIT




ECHO: ------ pass turn
CMD:PASS
CMD:MAIN
DUMP:DP,KEYWORDS

#EXIT

ECHO: ------ pass turn
CMD:PASS
DUMP:DP,KEYWORDS

EXIT

ECHO: go
CMD:ATTACK Res Tank
STEP:AUTOANSWER:CAPTURE
DUMP:DP,KEYWORDS,TRASH



