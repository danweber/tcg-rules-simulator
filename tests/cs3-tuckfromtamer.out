FIELD1:mon,CS3-Sorceryswan Tamer,CS3-Birdwatcherbill,CS3-Fairyfeatherlink,CS3-Chirpflare evo,CS3-Fairyfeatherlink
FIELD2:
GAME:P1 T3 MAIN 6
ECHO: ------ play,  should interrupt
EFFECT-TREE:DSE DEPTH 1 PROCESSING:normal evolve
EFFECT-TREE:INT DEPTH 1 SorcerySwan
GET-CHOICES:Choose:
GET-CHOICES:Activate SorcerySwan moving card
GET-CHOICES:Don't pay/activate
FIELD1:mon,CS3-Sorceryswan Tamer,CS3-Birdwatcherbill,CS3-Fairyfeatherlink,CS3-Chirpflare evo,CS3-Fairyfeatherlink
FIELD2:
GAME:P1 T3 MAIN 6
ECHO: -- choose to tuck
EFFECT-TREE:DSE DEPTH 1 PROCESSING:normal evolve
EFFECT-TREE:INT DEPTH 1 SorcerySwan
GET-CHOICES:Choose 1 target for moving card:
GET-CHOICES:Target FairyFeatherlink BATTLE-CS3-02
GET-CHOICES:Target ChirpFlare BATTLE-CS3-03
FIELD1:mon,CS3-Sorceryswan Tamer,CS3-Birdwatcherbill,CS3-Fairyfeatherlink,CS3-Chirpflare evo,CS3-Fairyfeatherlink
FIELD2:
GAME:P1 T3 MAIN 6
ECHO: -- choose chirp, and we evo for 4-1 or 3
EFFECT-TREE:NOTHING
GET-CHOICES:Choose 1 target for moving card:
FIELD1:mon,CS3-Sorceryswan Tamer,CS3-Birdwatcherbill,CS3-Fairyfeatherlink evo,CS3-Shieldwing,CS3-Fairyfeatherlink,CS3-Chirpflare
FIELD2:
GAME:P1 T3 MAIN 3
